loader/src/main/kotlin/net/weavemc/loader/util/FileManager.kt

```diff
@@ -12,16 +11,42 @@ internal object FileManager {
 
     fun getVanillaMinecraftJar(): File {
```

Comment:
A bit inconsistent and messy. Specifically with the use of file separators.

Reply:
Blame decencies :pensive: (i'll change it up)